docs/adding_software/adding_development_software.md

```diff
@@ -0,0 +1,80 @@
+# Adding software to `dev.eessi.io`
+
+## What is `dev.eessi.io`?
+
+`dev.eessi.io` is the development repository of EESSI. With it, developers can build and deploy non-production ready versions of their software to a CernVM-FS repository. This way, development version can easily be tested on systems where `dev.eessi.io` is available.
+
+## Adding software
+
+Using `dev.eessi.io` is similar to using EESSI's production repository `software.eessi.io`. Software builds are triggered by a bot listening to pull requests in GitHub repository (at the moment https://github.com/EESSI/dev.eessi.io). This repository is organised by project, where corresponding easystack files and easyconfig files are placed.
```

Comment:
We'll probably ends up with separate repos per project, so point to https://github.com/search?q=org%3AEESSI+dev.eessi.io&type=repositories ?